scp/cc/cpio/client_providers/private_key_fetcher_provider/src/private_key_fetcher_provider.h

```diff
@@ -28,6 +28,8 @@
 #include "error_codes.h"
 
 namespace google::scp::cpio::client_providers {
+    
+/// Response for fetching private key.
```

Comment:
Do we really need these changes in shared code?

scp/cc/cpio/client_providers/private_key_fetcher_provider/src/private_key_fetcher_provider.h
scp/cc/cpio/client_providers/private_key_fetcher_provider/src/private_key_fetcher_provider_utils.cc
scp/cc/cpio/client_providers/private_key_fetcher_provider/src/private_key_fetcher_provider_utils.h

are shared

Reply:
These are leftovers. I will undo them